server/src/main/java/org/elasticsearch/index/codec/PerFieldFormatSupplier.java

```diff
@@ -34,13 +33,12 @@
  * vectors.
  */
 public class PerFieldFormatSupplier {
-    public static final FeatureFlag USE_LUCENE101_POSTINGS_FORMAT = new FeatureFlag("use_lucene101_postings_format");
+    public static final FeatureFlag USE_DEFAULT_LUCENE_POSTINGS_FORMAT = new FeatureFlag("use_default_lucene_postings_format");
```

Comment:
Aligning this with #128236.    ( do not merge in main, will merge collapse this PR. Main will merge later and this will auto-merge - with minimal or no conflict)

Reply:
Additionally we disabled feature flag for a few days, since we're evaluating if default Lucene codec has regressions (before when we added the feature flag, there was noise, upgrading to Lucene 10.2 and enabling logsdb by default happened). We will re-enable on Monday.